src/models/setting.ts

```diff
@@ -88,12 +88,12 @@ export default {
     },
     changeSetting (state, { payload }) {
       const urlParams = new URL(window.location.href)
-      Object.keys(defaultSettings).forEach(key => {
+      for (const key in Object.keys(defaultSettings)) {
```

Comment:
Do we need `Object.keys` here? I think `for (let key in defaultSettings)` will work, because we know the object.

Reply:
done